Aur Install: could not find development headers and libraries

`makepkg` was giving me errors like this:

    In file included from /usr/include/KF5/KConfigWidgets/KCModule:1,
                 from /home/samuel/aur/krunner-pass/src/krunner-pass-1.3.0/config.h:23,
                 from /home/samuel/aur/krunner-pass/src/krunner-pass-1.3.0/config.cpp:24:
    /usr/include/KF5/KConfigWidgets/kcmodule.h:29:10: fatal error: kauthaction.h: No such file or directory
    #include <kauthaction.h>

I did the following things to resolve the problem:
- Added `DBus` to the find_package directive for Qt5
- Added `Auth` to the find_package directive for KF5
- Added `include_directories("/usr/include/KF5/KAuth/")` after the other `include` directives

I don't use CMake often so I don't know why I had to make these changes. It installed fine after that.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,8 @@ set (CMAKE_MODULE_PATH
      ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR} ${CMAKE_MODULE_PATH}
 )
 
-find_package (Qt5 ${QT_MIN_VERSION} REQUIRED CONFIG COMPONENTS Widgets Core Quick)
-find_package (KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS I18n Service Runner TextWidgets ConfigWidgets PlasmaQuick Notifications)
+find_package (Qt5 ${QT_MIN_VERSION} REQUIRED CONFIG COMPONENTS Widgets Core DBus Quick)
+find_package (KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS Auth I18n Service Runner TextWidgets ConfigWidgets PlasmaQuick Notifications)
 
 include(KDEInstallDirs)
 include(KDECMakeSettings)

